cppcache/src/TcrEndpoint.cpp

```diff
@@ -439,7 +440,7 @@ GfErrType TcrEndpoint::registerDM(bool clientNotification, bool isSecondary,
           "channel for endpoint %s",
           m_name.c_str());
       // setup notification channel for the first region
-      std::lock_guard<decltype(m_notifyReceiverLock)> guard(
+      std::lock_guard<decltype(m_notifyReceiverLock)> notifyReceiverLockGuard(
```

Comment:
Oh wow, that was a bad one. Confusing to rationalize when `guard` would be released.